packages/app/src/components/PageEditor/EmojiAutoCompleteHelper.js

```diff
@@ -1,149 +1,88 @@
-// import UpdateDisplayUtil from '~/client/util/codemirror/update-display-util.ext';
 
+import emojiData from 'emoji-mart/data/all.json';
+import { NimbleEmojiIndex } from 'emoji-mart';
+import UpdateDisplayUtil from '~/client/util/codemirror/update-display-util.ext';
```

Comment:
You don't need to import emojiData.
please show emoji-mart UI by the trigger input like `:a`